---
abstract: Precision medicine is transforming cancer treatment, with targeted therapy offering cures for previously untreatable disease. However, for most patients a targeted treatment cannot be identified, and these patients must rely on less effective conventional management. The limiting factor to wide application of precision oncology is our imperfect understanding of cancer biology. High-throughput omics studies have aimed to address this through unbiased discovery of cancer drivers, with limited success. Previous cancer ‘omics’ studies have mostly focused on individual genes in a single molecular modality (e.g., transcriptomics) at a time. However, cancer is driven by the dysregulation of multiple regulatory pathways, visible across different omics layers including genomics, transcriptomics, and proteomics. As genes work in concert across these diverse omics profiles to drive cancer, we reasoned that an integrative approach that considers multiple molecular modalities, in the context of multi-gene pathways, will yield the best understanding of cancer aetiology and development. Here we present InCRIMP, an analysis technique which integrates multiple molecular measurements, state-of-the-art network analysis techniques, and machine learning models to achieve comprehensive molecular dissection of high-throughput data, and unlock the true potential of molecular profiling to understand biology and disease. We demonstrate InCRIMP’s potential on a cohort of low-survival childhood cancers, but the technique is widely applicable to all domains where multi-modal molecular data must be distilled into biologically relevant and comprehensible modules. 
address:
  city: 
  country: 
  postcode: ""
  region: 
  street: 
all_day: false
author_notes:
- 
- 
authors:
- Adrian Salavaty
- Mark Pinese
date: "2022-11-01T00:00:00Z"
date_end: ""
doi: ""
event: AGTA
event_url: https://agtaconference.org/
featured: true
image:
  caption: 'Image credit: [**Adrian Salavaty**](https://asalavaty.com/author/adrian-salavaty/)'
  focal_point: ""
  preview_only: false
links:
- icon: image
  icon_pack: fas
  name: Poster
  url: talks/past_talks/incrimp_2022-2023/poster.pdf
- icon: globe
  icon_pack: fas
  name: AGTA
  url: https://agtaconference.org/
- icon: globe
  icon_pack: fas
  name: ICG
  url: https://www.icg2023.com.au/
location: AGTA 2022, Sunshine Coast, Queensland, Australia; ICG 2023, Melbourne Convention and Exhibition Centre (MCEC)
projects:
- 
publication_types:
- "1"
publishDate: "2022-11-01T00:00:00Z"
slides: 
summary: " "
tags:
- Cancer research
- Integrative model
- Multi-Omics
- Network analysis
title: InCRIMP; a versatile computational model for the integrative analysis of multi-omics data
url_code: 
url_dataset: ''
url_pdf: 
url_poster: ''
url_project: ""
url_slides: ""
url_source: ''
url_video: ''
---

## **Abstract**  
<div style="text-align: justify">
Precision medicine is transforming cancer treatment, with targeted therapy offering cures for previously untreatable disease. However, for most patients a targeted treatment cannot be identified, and these patients must rely on less effective conventional management. The limiting factor to wide application of precision oncology is our imperfect understanding of cancer biology. High-throughput omics studies have aimed to address this through unbiased discovery of cancer drivers, with limited success. Previous cancer ‘omics’ studies have mostly focused on individual genes in a single molecular modality (e.g., transcriptomics) at a time. However, cancer is driven by the dysregulation of multiple regulatory pathways, visible across different omics layers including genomics, transcriptomics, and proteomics. As genes work in concert across these diverse omics profiles to drive cancer, we reasoned that an integrative approach that considers multiple molecular modalities, in the context of multi-gene pathways, will yield the best understanding of cancer aetiology and development. Here we present InCRIMP, an analysis technique which integrates multiple molecular measurements, state-of-the-art network analysis techniques, and machine learning models to achieve comprehensive molecular dissection of high-throughput data, and unlock the true potential of molecular profiling to understand biology and disease. We demonstrate InCRIMP’s potential on a cohort of low-survival childhood cancers, but the technique is widely applicable to all domains where multi-modal molecular data must be distilled into biologically relevant and comprehensible modules.
</div>

### **Key words**
InCRIMP; Multi-omics network analysis; Module identification; Gene prioritization; Cancer risk genes; Cancer driver genes
